src/Spryker/Zed/AppMerchant/Business/Message/MessageSender.php

```diff
@@ -36,7 +36,7 @@ public function sendMerchantAppOnboardingStatusChangedMessage(
         MerchantTransfer $merchantTransfer,
         AppConfigTransfer $appConfigTransfer
     ): void {
-        $merchantOnboardingStatus = $merchantTransfer->getConfigOrFail()[AppMerchantConfig::MERCHANT_ONBOARDING_STATUS];
+        $merchantOnboardingStatus = $merchantTransfer->getConfig()[AppMerchantConfig::MERCHANT_ONBOARDING_STATUS];
```

Comment:
no default value?

Reply:
Wasn't there before. Only CI failures were fixed by this.